content/api/servo-assembly/arduino-commands.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Custom Arduino Commands
+---
+
+At the core of the servo assembly is an Arduino <TODO: model name>. By default
+the Arduino runs firmware created by SourceBots providing simple access to the
+attached servo shield, compatible ultrasound sensors and to the GPIO pins
+directly.
+
+You can also extend the firmware on the Arduino to support more features, for
+example if you have other types of sensor which require higher precision timing
+than can be achieved through the Python API.
+
+## Requirements
+
+As your new firmware will still need to be able to talk to the Python code
+running on the Raspberry Pi, you'll need to base it on SourceBots default
+firmware. You can get a copy of the source code from GitHub:
+
+```none
```

Comment:
Can we use `txt` here? I'm not even sure `none` is a valid highlight

Reply:
79fbd5626c6fc9709c32fb47431a2fd8b270b661